Source/GUI/MainWindow.cpp

```diff
@@ -74,6 +73,10 @@ void MainWindow::makeMenus()
 
   m_actMemoryViewer = new QAction(tr("&Memory Viewer"), this);
   m_actCopyMemory = new QAction(tr("&Copy Memory Range"), this);
+  m_actScanner = new QAction(tr("&Scanner"), this);
+  m_actScanner->setShortcut(QKeySequence("S"));
```

Comment:
> Bonus: The action has been assigned the `S` keyboard shortcut.

I'm actually against 'S' being a shortcut for this. It's easy to accidentally trigger, and also prevents the ability of pressing 'S' in the watchlist to snap to the next entry starting with 'S'.

I would be fine with Shift+S or some other combination rather than single key though.

Reply:
> ...
>
> I would be fine with Shift+S or some other combination rather than single key though.

I'm not particularly happy with `Shift+S`. That kind of shortcut is often used for saving, or saving a copy.

What if we go with `F3` (or some other number?)

Reply:
F3 is fine with me

Reply:
Whatever you decide in the Label under View lets add in parens the key.

Such that that item would say:
`Scanner (F3)`

Reply:
> I think you were thinking Ctrl+Shift+S or Ctrl+S.

Yes, but I wouldn't be surprised if there are other software that use `Shift+S` for similar actions.

Reply:
> Whatever you decide in the Label under View lets add in parens the key.
> 
> Such that that item would say: `Scanner (F3)`

Note that showing the keyboard shortcut in the action's label is something that Qt does automatically by virtue of adding a keyboard shortcut to the `QAction`.

In my system/desktop environment, it shows:
![image](https://github.com/aldelaro5/dolphin-memory-engine/assets/1853278/3f6208a1-3d21-4fb8-a976-d8fd41ce5483)

To do add the shortcut in parens, we'd have to do modify the action's label ourselves, and then try to find a way so that Qt does not show the keyboard shortcut at the right end.

Reply:
Ah ok, sounds good then. Lets not change it.